src/components/Account.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="flex">
+    <button
+      class="mt-2 mr-2 rounded-full bg-black hover:bg-purple-500 text-white p-2 px-4"
+      v-if="isStatic"
+      @click="initSdk(false)"
+    >
+      Connect Wallet Extension
+    </button>
+    <button
+      class="mt-2 mr-4 rounded-full bg-black hover:bg-purple-500 text-white p-2 px-4"
+      @click="modifySettings = !modifySettings"
+    >
+      <span v-if="isStatic">Modify Local Account</span>
+      <span v-if="!isStatic">Use Local Account</span>
```

Comment:
You can use v-else here :)

Reply:
yeah its copied from legacy, I'll adjust